[SS-2218] Email UI Fixes

utkarsh14 opened the pull request:
## [SS-2218] Email UI Fixes

## Ticket

Fixes: https://idinsight.atlassian.net/browse/SS-2218

## Description, Motivation and Context

Email UI fixes:

- Standard UI changes for bordered table, custom button
- Changed email time to be in 12 hour format
- Fixed email delivery report UI, added filters on table
- Fetch enumerator level report when user switches date in delivery report.

## How Has This Been Tested?
Local

## Checklist:

This checklist is a useful reminder of small things that can easily be forgotten.
Put an `x` in all the items that apply and remove any items that are not relevant to this PR.

- [ ] My code follows the style guidelines of this project
- [ ] I have reviewed my own code to ensure good quality
- [ ] I have tested the functionality of my code to ensure it works as intended
- [ ] I have resolved merge conflicts
- [ ] I have updated the automated tests (if applicable)
- [ ] I have written [good commit messages][1]
- [ ] I have updated the README file (if applicable)
- [ ] I have updated affected documentation (if applicable)

[SS-2218]: https://idinsight.atlassian.net/browse/SS-2218?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

utkarsh14 committed:
Email UI Fixes

jeenut27 reviewed (approved):
This looks good. I was testing the email UI and noticed a couple of things:

1. On the Email Delivery Report popup, if a schedule doesn't have any emails - the table continues to show the data from the previously selected schedule
2. The Trigger Time on manual triggers screen still shows a full date format. We had discussed updating this to date and time for uniformity. The View Recipients button here can also be labelled view delivery report for uniformity.

Approving! Feel free to merge

utkarsh14 committed:
Fix Email Reports and Triggers

utkarsh14 commented:
<img width="1281" height="277" alt="image" src="https://github.com/user-attachments/assets/33e74eea-62ab-46e8-b62c-70b13ea97115" />

Fixed trigger dae times and view recipient button.


> schedule doesn't have any emails - the table continues to show the data from the previously selected schedule

Both tables had same data and first fetch was not working i have added an use effect to fix it.
